docs/features.md

```diff
@@ -0,0 +1,53 @@
+# Features
+
+Stratos UI provides the feature set outline below. Some of these are illustrated in the [Screenshots gallery](images/screenshots/README.md).
+
+* Authentication
+   * User authentication using a configured UAA instance
+   * Two supported personas - Console admins and Console users
+
+* Endpoint Management
+   * The ability for an administrator to register one or more Cloud Foundry clusters to be managed through the UI.
+   * Ability for users to connect to Cloud Foundry clusters with their own credentials so that they receive the correct level of access to these clusters
```

Comment:
Think we should add that some features are conditional on the deployment mode. For instance this isn't available when deployed in CF.